python/katana/dynamic_bitset.pyx

```diff
@@ -67,3 +70,65 @@ cdef class DynamicBitset:
     def count(self):
         """Counts how many bits are set in the bitset."""
         return self.underlying.count()
+
+    @property
+    def address(self):
+        return <uintptr_t>&self.underlying
+
+
+# Number wrappers
+
+from libc.stdint cimport uint8_t, uint64_t, uintptr_t
+
+import ctypes
+
+from katana.dynamic_bitset cimport CDynamicBitset, DynamicBitset
+
+from katana.numba_support.wrappers import SimpleNumbaPointerWrapper
+
+DynamicBitset_numba_type_wrapper = SimpleNumbaPointerWrapper(DynamicBitset, override_module_name=__name__)
+DynamicBitset_numba_type = DynamicBitset_numba_type_wrapper.Type
+
+
+cdef uint64_t size(CDynamicBitset *self) nogil:
+    return self.size()
+
+DynamicBitset_numba_type_wrapper.register_method(
+    "size",
+    ctypes.CFUNCTYPE(ctypes.c_uint64, ctypes.c_void_p),
+    addr=<uintptr_t>&size,
+)
+
+
+cdef uint8_t get_item(CDynamicBitset *self, uint64_t key) nogil:
+    if key < 0 or key >= self.size():
+        raise IndexError(key)
+
+    return <uint8_t>self.test(key)
+
+DynamicBitset_numba_type_wrapper.register_method(
+    "get_item",
```

Comment:
minor: we could use __name__ in case IDEs are not able to connect the string to the name of a function/method.

Reply:
This is an interesting idea. I will explore it later. I'm not sure it will work because cdef functions in Cython are not full objects in the sense that functions are in Python.